data-serving/data-service/api/openapi.yaml

```diff
@@ -541,6 +573,12 @@ components:
         application/json:
           schema:
             $ref: '#/components/schemas/BatchValidateCaseResponse'
+    "207BatchUpsertCaseResponse":
```

Comment:
Wow, I don't think I've seen status 207 correctly used in an HTTP API before. Chapeau 🎩

Reply:
It's a bit elusive, isn't it? I'm hoping the final state here will taken better advantage yet.